op-acceptance-tests/justfile

```diff
@@ -0,0 +1,25 @@
+# Default recipe - runs the acceptance tests with default parameters
+default:
+    @just acceptance-test
+
+# Run acceptance tests against a devnet
+acceptance-test devnet="simple" gate="localnet":
+    #!/usr/bin/env bash
+    set -euo pipefail
+
+    # First run the appropriate devnet from the kurtosis-devnet directory if needed
+    # we ignore failures here because the devnet may already be running
+    (cd ../kurtosis-devnet && just {{ devnet }}-devnet) || true
+
+    # Get the absolute path to the repository root
+    REPO_ROOT=$(cd .. && pwd)
+
+    # Run op-acceptor with the repository mounted at the correct Go module path
+    docker run \
+        -v "$(pwd)/acceptance-tests.yaml:/acceptance-tests.yaml" \
+        -v "$REPO_ROOT:/go/src/github.com/ethereum-optimism/optimism" \
+        us-docker.pkg.dev/oplabs-tools-artifacts/images/op-acceptor:v0.1.1
```

Comment:
for convenience we probably should make the image an overridable variable

So we can do `just ACCEPTOR_IMG=... acceptance-test`

Reply:
Good thinking.
I made this change, but it's slightly different (perhaps you intended to type this in fact?):
`ACCEPTOR_IMAGE=... just acceptance-test`

Reply:
yeah either way works. `just VAR=val ...` sets the just variable directly.
`VAR=val just ...` sets the environment, which the justfile uses to set the default value foor the just variable